jekyll/_cci2/contexts.md

```diff
@@ -48,8 +48,51 @@ workflows:
 
 To rerun a job and use the context, it **must** be rerun from the Workflows page of the CircleCI application.
 
+## Securing a Context
+
+CircleCI enables you to secure secrets at run time by adding security groups to contexts. Only Org Admins may add *security groups* to a new or existing context. Security groups are definied as LDAP groups or GitHub teams. After the security group is added to a context, only members of that security group may access or use the environment variables of the secure context. 
```

Comment:
secure secrets -> Should we say secrets or environment variables?

Reply:
thanks, changed to 'secret environment variables' and 'organizaiton administrators'